lib/tasks/migrations/migrations.rake

```diff
@@ -74,6 +74,43 @@ namespace :migrations do
     exit has_encountred_issue
   end
 
+  task :rooms, [] => :environment do |_task, _args|
+    has_encountred_issue = 0
+
+    Room.select(:uid, :name, :bbb_id, :last_session, :user_id)
+        .find_each(batch_size: COMMON[:batch_size]).each do |r|
+          params = { room: { friendly_id: r.uid,
+                             name: r.name,
+                             meeting_id: r.bbb_id,
+                             last_session: r.last_session.to_datetime,
+                             owner_email: r.user.email,
+                             owner_provider: r.user.provider } }
```

Comment:
Provider is expected to be 'greenlight' may fail the rooms migration,

Reply:
Won't that be removed for production?

Reply:
AFAIK it's production ready,
Should have @farhatahmad input on this,

Reply:
I believe we need to remove the specification of the room provider,
However, besides failing to migrate rooms this will also fail because the room owner is `r.owner` not `r.user` on v2,

Reply:
I'm pretty confident I tested `r.user` but I can double check.

Reply:
Yeah ok I will change to owner. It's weird I'm sure I tested it